app/test_app/functional/api/test_auth_api.py

```diff
@@ -505,3 +507,45 @@ def test_token_auth(test_client, authed_sempo_admin_user, role, tier, status_cod
                                ),
                                content_type='application/json', follow_redirects=True)
     assert response.status_code == status_code
+
+
+@pytest.mark.parametrize("user, is_activated, is_disabled, message", [
+    (False, None, None, 'user not found'),
+    (True, False, None, 'user not activated'),
+    (True, True, True, 'user has been disabled'),
+])
+def test_token_auth_errors(test_client, authed_sempo_admin_user, user, is_activated, is_disabled, message):
+    """
+    GIVEN a Flask application
+    WHEN the '/api/auth/check/token/' api is requested (GET)
+    THEN check the response is only returned for superadmin
+    """
+
+    if user:
```

Comment:
I think this probably makes more sense as two tests (one real user one fake), rather than doing an `if` and parameterisation. Also it'd be good to have one where the held role is set to `admin` and `supervendor` just to cover our bases

Reply:
good point! iterated